File: src/chapter04/language/Rechteck.java
package chapter04.language;

import java.util.Iterator;

public class Rechteck {

	public static void main(String[] args) {	
		int l = 5;
		int b = 10;		

		int umfang = l * 2 + b * 2;
		int flaeche = l * b;	

		System.out.printf("Das Rechteck mit Länge %s und Breite %s hat: %n", l, b);
		System.out.println("Umfang = " + umfang);
		System.out.println("Fläche = " + flaeche);
	}

}

File: lab/04-classes/rechteck.py

class Rechteck:

    def __init__(self, laenge, breite):
        self.laenge = laenge
        self.breite = breite

    def umfang(self):
        return 2 * (self.breite + self.laenge)

    def flaeche(self):
        return self.laenge * self.breite

    def print(self):
        print("Rechteck mit Länge:", self.laenge, "und Breite:", self.breite)
        print("- Umfang =", self.umfang())
        print("- Fläche =", self.flaeche())


if __name__ == '__main__':
    r = Rechteck(3,7)
    r.print()
